File: TaskStack/Models/Task.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace TaskStack
{
    public class Task
    {
        [Key]
        public int Id { get; set; }
        public string TMName { get; set; }
        public string BriefDesc { get; set; }
        public string DueDate { get; set; }
        public bool isCompleted { get; set; }
    }
}

File: TaskStack/ClientApp/src/app/Task.ts
export interface Task{
    id?:number;
    tmName?:string;
    briefDesc?:string;
    dueDate?:string;
    isCompleted?:boolean;
}

export class Convert {
    public static toTask(json: string): Task {
        return JSON.parse(json);
    }
    public static taskToJson(value: Task): string {
        return JSON.stringify(value);
    }
}
